CASMTRIAGE-7099-1.5 port of CSM 1.4 full system power off/on procedures to CSM 1.5 (#5442)
Cray-HPE/docs-csm: operations/power_management/Prepare_the_System_for_Power_Off.md

   To prevent this issue from happening, remove stale `ssh` host keys from `/root/.ssh/known_hosts` before running the `sat` command.

> 1. Check certificate expiration deadlines to ensure that a certificate won't expire while the system is powered off.

Reviewer: ```suggestion
1. Check certificate expiration deadlines to ensure that a certificate will not expire while the system is powered off.
```

Author: fixed

    There is no method to prevent new sessions from being created as long as the service APIs are accessible on the API gateway.

1. Notify users and operations staff about the upcoming full system power off.

>    The notification method will vary by system, but might be email, messaging applications, `/etc/motd` on UANs, `wall` commands on UANs, etc.

Reviewer: ```suggestion
   The notification method will vary by system, but might be email, messaging applications, `/etc/motd` on UANs, `wall` commands on UANs, and so on.
```

Author: fixed

       scontrol update NodeName=ALL State=DRAIN Reason="Shutdown"
       ```

    1. For PBS Professional, see the `qstat` and `qmgr` man pages.

>        Below is an example to list the available queues, disable a specific queue named `workq`, and check

Reviewer: ```suggestion
       The following is an example to list the available queues, disable a specific queue named `workq`, and check
```

Author: Fixed for both Slurm and PBS examples.
